src/exo0 - Composing with pipe and flow/exo0.exercise.ts

```diff
@@ -63,10 +64,10 @@ export const ifThenElse =
 // Using `pipe` and `ifThenElse`, write the function that computes the next step in the Collatz
 // sequence.
 
-export const next: (value: number) => number = unimplemented;
+export const next = (value: number) => pipe(value, isOddP, ifThenElse(() => value * 3 + 1, () => value/2));
 
 // Using only `flow` and `next`, write the function that for any given number
 // a_n from the Collatz sequence, returns the number a_n+3 (i.e. the number
 // three steps ahead in the sequence).
 
-export const next3: (value: number) => number = unimplemented;
+export const next3 = (value: number) => flow(next, next, next)(value);
```

Comment:
Given my previous comment, you should be able to rewrite this one in an even more concise way :)